### PR TITLE
Microsoft.AspNetCore.Mvc.Razor.Precompilation.Tools does not need to target desktop

### LINKED ISSUE #19: Microsoft.AspNetCore.Mvc.Razor.Precompilation.Tools does not need to target desktop
Per @NTaylorMullen, the tool doesn't need to be cross compiled. `dotnet` always calls the netcoreapp1.0 version of the tool.

### PR DESCRIPTION
**Pull request by pranavkm**
Fixes #19

### CODE REVIEW

File: src/Microsoft.AspNetCore.Mvc.Razor.Precompilation.Tools/project.json [hunk 1]
```diff
@@ -45,7 +45,6 @@
         },
         "System.Runtime.Serialization.Primitives": "4.3.0-*"
       }
-    },
-    "net451": {}
```
Comment (pranavkm) [inline src/Microsoft.AspNetCore.Mvc.Razor.Precompilation.Tools/project.json]: Tried it out manually. The CLI invokes the tool correctly for `net451` only and `netcoreapp1.x` only projects. Not sure if we'd want to add a functional test to make sure it runs in these cases.

Comment (NTaylorMullen) [inline src/Microsoft.AspNetCore.Mvc.Razor.Precompilation.Tools/project.json]: Could be a follow up. We do it for Razor Tooling.

**Comment by NTaylorMullen** [comment]
:shipit:

**Comment by NTaylorMullen** [comment]
:shipit:

**Pull request merged**